### ISSUE TITLE
Rearrange the item forever in the end

### ISSUE THREAD

**Issue by THEb0nny** [topjohnwu/Magisk#535]
Change the point of issuing rights permanently elsewhere because it is not convenient to rearrange each time.
When you give the right for good, you expose it once.
I already saw such a request and you gave it a go, but there was no change. You seem to have forgotten.
<string-array name="allow_timeout">
        **<item>@string/forever</item>**
        <item>@string/once</item>
        <item>@string/tenmin</item>
        <item>@string/twentymin</item>
        <item>@string/thirtymin</item>
        <item>@string/sixtymin</item>
        **here**
</string-array>

**Comment by neekless** [comment]
I agree with this. Currently the default option when an app ask for root access is to grant "forever". If I only want to grant it once I have to manually select "once", and have to do this every time I want to grant it once.

It would make more sense for it to default to "once" so that it's easier and quicker to grant once, and if someone wants to grant root access forever they should have to actively choose it. Plus they don't have to do it again after granting forever so the manual selection is less of a hassle for them.

**Comment by dioni21** [comment]
+1

**Comment by THEb0nny** [comment]
Perhaps once the second point, too, will interfere ... Although I'm right for 1 minute.

**Comment by THEb0nny** [comment]
The developer looked, but did not react. A new version of Manager has been released upon request, but there is no solution.
